### DISCUSSION TITLE
Is DevOps Dead...

### DISCUSSION THREAD

**Discussion by mcgonagle** [cdfoundation/sig-best-practices#9] [General]
The DevOps community in Boston is dwindling. Are we at DevOps 3.0? Or is it just covid?

**Comment by tequilarista** [comment]
From BoF discussions:

Key to this discussion is understanding what DevOps is and is not. 

One a somewhat related note, one of the quotes from cdCon keynote was: “There are no best practices, anything goes, just wash your hands” — is this just a different variant of “DevOps must be dead, if there isn’t any guidance”

DORA research clearly shows there are best practices, and that they can be measured.  Organizations not understanding or balking at the amount of investment required can certainly be a roadblock.

Perhaps a refocus on multi-directional approach to DevOps — one vector “what is the target business ROI”, another vector is “what are the practical technical needs and which priority should they be implemented”
